array-destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,17 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPrice: 1.0 },
   { itemName: "Hash Brown", quantity: 4, unitPrice: 0.4 },
 ];
+
+function receiptForOrder(order) {
+  let unitTotalPrice = 0;
+  let total = 0;
+  order.forEach(({ itemName, quantity, unitPrice }) => {
+    unitTotalPrice = unitPrice * quantity;
```

Comment:
There is no need to create the variable on line 11 if you are assigning it on line 14.

You can change this to define the variable on line 14